ansible/roles/master/templates/haproxy.cfg.j2

```diff
@@ -0,0 +1,73 @@
+kip to content
```

Comment:
Noted, I'll remove keepalived/haproxy into separate roles and submit a separate PR.